deployments/examples/ocis_oc10_backend/oc10/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM owncloud/server:latest
```

Comment:
Codacy found an issue: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6324067)

Reply:
use 10.5.0 which is the latest.

As soon as we have a 10.6.0-something we should update

Reply:
what do you think about taking the tag owncloud/server:10 which refers always to the latest image in our oc 10 tree?